tests/test_path.py

```diff
@@ -0,0 +1,24 @@
+import wannierberri as wb
+import pythtb as ptb
+import numpy as np
+
+def test_path():
+    def HaldanePTB(delta,t1,hop2,phi):
+        lat=[[1.0,0.0],[0.5,np.sqrt(3.0)/2.0]]
+        orb=[[1./3.,1./3.],[2./3.,2./3.]]
+        haldane=ptb.tb_model(2,2,lat,orb)
+        haldane.set_onsite([-delta,delta])
+        haldane.set_hop(t1, 0, 1, [ 0, 0])
+
+        return haldane
+
+
+    haldane = HaldanePTB(2,1,1/3,np.pi/10)
+    syst=wb.System_PythTB(haldane,berry=True,morb=True)
+
```

Comment:
We already have a fixture 

 
 ​system_Haldane_PythTB​

Is there a reason not to use it here?

Reply:
No, I didn't know it. I'll fix it soon.